source/features/tags-on-commits-list.tsx

```diff
@@ -139,7 +139,7 @@ async function init(): Promise<void | false> {
 
 void features.add(import.meta.url, {
 	include: [
-		pageDetect.isRepoCommitList,
+		pageDetect.isCommitList,
```

Comment:
Unrelated change, the DOM for pagination is different and it’s unlikely to find tagged commits in PRs, so not worth it for the vast majority of cases. This feature is not optimized

Reply:
I am not sure what you mean by that. Can you elaborate the change? This feature handles rendering in repo and PR commits.